pyproject.toml

```diff
@@ -188,6 +188,7 @@ filterwarnings= [
   # https://github.com/python-trio/trio/issues/3053
   "ignore:The `hash` argument is deprecated in favor of `unsafe_hash` and will be removed in or after August 2025.",
 ]
+asyncio_default_fixture_loop_scope = "function"
```

Comment:
I think this comes from `pytest-asyncio`, but don't we use AnyIO's pytest plugin?

Reply:
It does come from `pytest-asyncio`, and evidently it is used via some means as there are warnings before this change and not after. But I am going to revert this change as this option is not available for the minimum dependencies run, and I can tolerate a warning.